### PR TITLE
[Emu3.5] Support Emu3.5 Inference with vLLM

### PR DESCRIPTION
**Pull request by zhaoyinglia**
### PR Category

Inference
### PR Types

New Models
### PR Description

Add vLLM backend support to enable efficient inference for Emu3.5 AR.
New features include:
- Batch scheduler between cond_input and uncond_input.
- Customized logits processor: ClassifierFreeGuidanceLogitsForVisualTokenProcessor.

DONE List:
- [x] Update FlagScale vllm backend to tag-0.11.0
- [x] support Emu3.5 offline inference with v1 engine

Example:
```
[v1/core/sched/batch_manager.py:247] Set customized hw: 55 x 73 for request 0_cfg_0
[v1/core/sched/batch_manager.py:247] Set customized hw: 55 x 73 for request 1_cfg_0
INFO 11-19 15:56:22 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 21.7 tokens/s, Avg generation throughput: 103.4 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 0.9%, Prefix cache hit rate: 0.0%
INFO 11-19 15:56:32 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 123.8 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 1.7%, Prefix cache hit rate: 0.0%
INFO 11-19 15:56:42 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 123.3 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 2.4%, Prefix cache hit rate: 0.0%
INFO 11-19 15:56:52 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 123.1 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 3.2%, Prefix cache hit rate: 0.0%
INFO 11-19 15:57:02 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 122.6 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 3.9%, Prefix cache hit rate: 0.0%
INFO 11-19 15:57:12 [v1/metrics/loggers.py:127] Engine 000: Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 122.2 tokens/s, Running: 2 reqs, Waiting: 0 reqs, GPU KV cache usage: 4.7%, Prefix cache hit rate: 0.0%


Processed prompts:  50%|█████     | 1/2 [01:06<01:06, 66.29s/it, est. speed input: 3.70 toks/s, output: 61.50 toks/s]
Processed prompts: 100%|██████████| 2/2 [01:06<00:00, 66.29s/it, est. speed input: 3.94 toks/s, output: 123.01 toks/s]
Processed prompts: 100%|██████████| 2/2 [01:06<00:00, 33.14s/it, est. speed input: 3.94 toks/s, output: 123.01 toks/s]
[2025-11-19 15:57:18,505 FlagScale logger.py:25 INFO] ----------------------------------------
[2025-11-19 15:57:19,608 FlagScale logger.py:25 INFO] >>> 📷[OUTPUT-0][image]: saved to outputs/emu3p5_one_image_generation/000/output_000_0.png
```

output_000_0

To-do / Verification List:
- [ ] DataParalel
- [ ] ChunkPrefill
- [ ] Continue Batching
- [ ] add emu3 v1 version

### CODE REVIEW

File: examples/emu3.5/conf/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```
+# install FlagScale(vLLM)
+git clone https://github.com/flagos-ai/FlagScale.git
+cd FlgScale
+python tools/patch/unpatch.py --backend vllm
+
+
+
+# git clone `src/` from Emu3.5
+git clone --no-checkout https://github.com/baaivision/Emu3.5.git tmp_repo
```
Comment (cyber-pioneer) [inline examples/emu3.5/conf/README.md:22]: better add corresponding branch or commit id?

Comment (zhaoyinglia) [reply examples/emu3.5/conf/README.md:22]: DONE.
